online-shop/src/util/CustomModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+interface IModalProps {
+    title: string;
+    status: boolean;
+    closeModal: () => void;
+    confirmModal: () => void;
+}
+
+export const CustomModal: React.FC<IModalProps> = (props: IModalProps) => {
+
+    let classes = "modal";
+
+    if (props.status) {
+        classes += " is-active";
+    }
+
+    return (
+        <div className={classes}>
```

Comment:
For such simple cases, typically you would just do a ternary expression in-line: `className={'modal' + (status ? ' is-active' : '')}`.